### ISSUE TITLE
Save video to... dialog shows for a milisecond

### ISSUE THREAD

**Issue by SimonC-24** [kakajuro/tidytube#15]
Save video to... dialog shows for a milisecond... then closes by itself, therefore not able to select my playlist 
![image](https://github.com/user-attachments/assets/210b3800-f141-4e41-babb-cf219967655f)

**Comment by kakajuro** [comment]
Yeah this was something I noticed, working on a fix right now. What browser are you using by the way?

**Comment by SimonC-24** [comment]
Sorry, forgot to mention that... 
![image](https://github.com/user-attachments/assets/9ab52fa7-ad48-439e-8738-c4c9a0bdfbc4)

**Comment by kakajuro** [comment]
Fixed! Will be live in the next update, can't say exactly when that will be at this point though.

**Comment by SimonC-24** [comment]
Thanks for the update! Now Save video to... dialog shows. Keep good work man.
